mcp-servers/llamastack/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mcp-llamastack-server
+spec:
+  selector:
+    matchLabels:
+      app: mcp-llamastack-server
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mcp-llamastack-server
+    spec:
+      serviceAccountName: ocp-mcp
+      containers:
+      - name: mcp-llamastack-server
+        image: quay.io/eoconnor/mcp-llamastack-server:latest
+        ports:
+          - containerPort: 8000
+            protocol: TCP
+          - containerPort: 8080
+            protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mcp-llamastack-server
+  labels:
+    app: mcp-llamastack-server
+spec:
+  selector:
+    app: mcp-llamastack-server
+    deployment: mcp-llamastack-server
+  ports:
+  - port: 8000
+    targetPort: http
+    protocol: TCP
+---
+apiVersion: v1
```

Comment:
The llamastack server will not require a ServiceAccount or RoleBinding as it doesn't have to update the OpenShift Cluster. We can remove line 40-56.